python/ray/_private/worker.py

```diff
@@ -1924,8 +1936,9 @@ def print_to_stdstream(data):
             batches = [data]
         sink = sys.stdout
 
+    # Ignore the prefix if the logging config is set.
```

Comment:
this comment should be moved to the caller

Reply:
removed https://github.com/ray-project/ray/pull/47703/commits/0556458613658c85a7619ff90e9a8f450c143ebc